test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -968,6 +983,31 @@ public void close() throws IOException {
 
     public static final String TRANSPORT_CLIENT_PREFIX = "transport_client_";
 
+    private class TransportServiceStartupCountDown {
+        private final int initialCount;
+        private final CountDownLatch countDownLatch;
+
+        TransportServiceStartupCountDown(int count) {
+            initialCount = count;
+            countDownLatch = new CountDownLatch(count);
+        }
+
+        void countDown() {
+            logger.info("transport service started: {} of {} remaining", countDownLatch.getCount() - 1, initialCount);
```

Comment:
I wonder if INFO logging is too verbose here. If start up fails, we should already have failures. So what's the benefit of this?

Reply:
This really came about from attempts to track down the need for https://github.com/elastic/elasticsearch/blob/31c1d7de21428140290166209f29d7ef1a20382f/test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java#L599. We discussed and decided not to use a countdown at all - see 31c1d7d.